java/trace/src/main/scala/org/typelevel/otel4s/java/trace/Scope.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.java.trace
+
+import io.opentelemetry.context.{Context => JContext}
+import io.opentelemetry.api.trace.{Span => JSpan}
+import org.typelevel.otel4s.trace.SpanContext
+
+sealed trait Scope
+
+object Scope {
+  val root: Scope = Root(JContext.root)
```

Comment:
Some `Scope` needs to be exposed.  This was the best I could do to hide it.  We could consider another constructor based on a `JContext` and wrap that in `Root`.

I thought about making `JContext` the environment in the public API and lensing it to `Scope`, but it doesn't round trip.

Reply:
As is emerging in #125, we probably want our own `Context` for propagation purposes.  We could retrofit that later if we like the general direction of this?